docs/docs/cmd/aad/user/user-get.mdx

```diff
@@ -74,57 +77,144 @@ Get information about the currently logged in user using the UserName token
 m365 aad user get --userName "@meUserName"
 ```
 
+Get information about a user with the manager's details
+
+```sh
+m365 aad user get --userName AarifS@contoso.onmicrosoft.com --withManager
+```
+
 ## Response
 
+### Standard response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  {
+    "businessPhones": ["+32499-99-99-99"],
+    "displayName": "John Doe",
+    "givenName": "John",
+    "jobTitle": "Sales Manager",
+    "mail": "john.doe@contoso.com",
+    "mobilePhone": "+32499-99-99-99",
+    "officeLocation": "Vosselaar",
+    "preferredLanguage": "nl-BE",
+    "surname": "Doe",
+    "userPrincipalName": "john.doe@contoso.com",
+    "id": "990e2425-f595-43bc-85ed-b89a44093793"
+  }
+  ```
+
+  </TabItem>
+  <TabItem value="Text">
+
+  ```text
+  businessPhones   : ["+32499-99-99-99"]
+  displayName      : John Doe
+  givenName        : John
+  id               : 990e2425-f595-43bc-85ed-b89a44093793
+  jobTitle         : Sales Manager
+  mail             : john.doe@contoso.com
+  mobilePhone      : +32499-99-99-99
+  officeLocation   : Vosselaar
+  preferredLanguage: nl-BE
+  surname          : Doe
+  userPrincipalName: john.doe@contoso.com
+  ```
+
+  </TabItem>
+  <TabItem value="CSV">
+
+  ```csv
+  displayName,givenName,mail,preferredLanguage,surname,userPrincipalName,id
+  John Doe,John,john.doe@contoso.com,nl-BE,Doe,john.doe@contoso.com,990e2425-f595-43bc-85ed-b89a44093793
+  ```
+
+  </TabItem>
+  <TabItem value="Markdown">
+
+  ```md
+  # aad user get --userName "john.doe@contoso.com"
+
+  Date: 16/02/2023
+
+  ## John Doe (990e2425-f595-43bc-85ed-b89a44093793)
+
+  Property | Value
+  ---------|-------
+  displayName | John Doe
+  givenName | John
+  mail | john.doe@contoso.com
+  preferredLanguage | nl-BE
+  surname | Doe
+  userPrincipalName | john.doe@contoso.com
+  id | 990e2425-f595-43bc-85ed-b89a44093793
+  ```
+
+  </TabItem>
+</Tabs>
+
+### `withManager` response
+
+When we make use of the option `withManager` the response will differ. 
+
 <Tabs>
   <TabItem value="JSON">
 
   ```json
   {
-    "businessPhones": [],
+    "businessPhones": ["+32499-99-99-99"],
     "displayName": "John Doe",
-    "givenName": null,
-    "jobTitle": null,
-    "mail": null,
-    "mobilePhone": null,
-    "officeLocation": null,
-    "preferredLanguage": null,
-    "surname": null,
+    "givenName": "John",
+    "jobTitle": "Sales Manager",
+    "mail": "john.doe@contoso.com",
+    "mobilePhone": "+32499-99-99-99",
+    "officeLocation": "Vosselaar",
+    "preferredLanguage": "nl-BE",
+    "surname": "Doe",
     "userPrincipalName": "john.doe@contoso.com",
-    "id": "74a3b772-3122-447b-b9da-10895e238219"
+    "id": "990e2425-f595-43bc-85ed-b89a44093793",
+    "manager": {
+      "displayName": "Jane Doe",
+      "userPrincipalName": "jane.doe@ordidev.onmicrosoft.com",
+      "id": "eb77fbcf-6fe8-458b-985d-1747284793bc",
+      "mail": "jane.doe@ordidev.onmicrosoft.com"
```

Comment:
It's kind of nitpicking, but it doesn't really make sense that the manager has another mail domain than the employer.